src/config.rb

```diff
@@ -70,6 +72,18 @@ def configure_file
       File.join(repo_path, 'configure.yaml')
     end
 
-    attr_reader :cli
+    def domain_answers_file
```

Comment:
In my answer validator, I need to know the relative paths for each of the `Configure` commands inside the answer `directory`. So I have already added a relative path function where required. As a result we don't need three separate method to handle each of the sections. One methods should be sufficient.